volttron/platform/dbutils/postgresqlfuncts.py

```diff
@@ -350,7 +350,7 @@ def get_topic_meta_map(self):
             'SELECT topic_id, metadata '
             'FROM {}').format(Identifier(self.meta_table))
         rows = self.select(query)
-        meta_map = {tid: jsonapi.loads(meta) if meta else None for tid, meta in rows}
+        meta_map = {tid: meta if meta and type(meta) == dict else jsonapi.loads(meta) if meta else None for tid, meta in rows}
```

Comment:
`type(meta) == dict` should be `isinstance(meta, dict)`

Reply:
Either works...

```
>>> a = {"alpha": "beta"}
>>> type(a)
<class 'dict'>
>>> type(a) == dict
True
>>> isinstance(a, dict)
True
```

unless I am missing something in the above code?

Reply:
It depends if you want it to work with subclasses of `dict` including `defaultdict` and `OrderedDict`.

```Python
>>> class Foo(dict):
...     pass
... 
>>> type(Foo())
<class '__main__.Foo'>
>>> isinstance(Foo(), dict)
True
>>> from collections import OrderedDict, defaultdict
>>> a = OrderedDict()
>>> b = defaultdict()
>>> type(a)
<class 'collections.OrderedDict'>
>>> type(b)
<class 'collections.defaultdict'>
>>> type(b) == dict
False
>>> isinstance(a, dict)
True
>>> isinstance(b, dict)
True
>>> 
```

Reply:
@ntenney as a future proofing please use isinstance rather than type(meta)